src/lib/components/02-components/callout/callout.tsx

```diff
@@ -108,20 +112,32 @@ export default function Callout({
           >
             {header}
           </Typography>
-          <Typography variant="body-2" className={colorSchema[color].content}>
-            {content}
-          </Typography>
-          <Button
-            animated
-            as="link"
-            href={link.url}
-            component={LinkElement}
-            endIcon={<ArrowRightIcon />}
-            color={colorSchema[color].button}
-            className="mt-md justify-center md:w-fit"
-          >
-            {link.title}
-          </Button>
+          {/* Content */}
+          {ContentElement ? (
+            <ContentElement
+              variant="body-2"
+              className={cn(colorSchema[color].content, 'text-body-2')}
```

Comment:
For the callout, the text should be `text-body-2`, but the [HTMLContent](https://github.com/fourkitchens/forcepoint-nextjs-page-router/blob/74349166a7e25bc223072a308fb2816888b2dc52/src/components/wysiwyg/html-content.tsx#L90) component only supports `text-body-3` and I don't know how override it (besides modifying the html component)